docs/01-overview/02-main-areas/README.md

```diff
@@ -1,28 +1,38 @@
 ---
 title: Main areas
-type: Overview
 ---
 
 Kyma consists of these main areas and components:
 
 ![areas](./assets/kyma-areas.svg)
 
-### Application connectivity
+### What is Application Connectivity?
 
 - Simplifies and secures the connection between external systems and Kyma
 - Registers external events and APIs and simplifies the API usage
 - Provides asynchronous communication with services and Functions deployed in Kyma through events
 - Manages secure access to external systems
 - Provides monitoring and tracing capabilities to facilitate operational aspects
 
-### Service consumption
+### What is API exposure?
```

Comment:
```suggestion
### What is API Exposure?
```
I'm not sure about the suggestion I'm making; however, all the other headlines are capitalized.

Reply:
I agree, we should be consistent one way or the other. 
The same is true for the subtab titles, we should align whether we want capitalisation or not.